a10_octavia/controller/worker/tasks/virtual_server_tasks.py

```diff
@@ -48,7 +48,10 @@ def set(self, set_method, loadbalancer, flavor_data=None, **kwargs):
         elif str(desc).isspace() or not str(desc):
             desc = ""
         else:
-            desc = '"{}"'.format(desc)
+            if len(desc) > 61:
```

Comment:
Why is this not a check on `len(desc) >= 63` as the limit is 63? `len(desc) > 62` also works.

Reply:
Lengths of 62 are valid. And the current condition makes it so they are not.

Reply:
I have trimmed it 61 because in the next line we are adding two characters (double quotes); making the string length to 63.

https://github.com/a10networks/a10-octavia/blob/master/a10_octavia/controller/worker/tasks/virtual_server_tasks.py#L51.